tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -340,7 +340,28 @@ def make_csv_dataset_v2(
   tuple that corresponds to a batch of CSV rows. The features dictionary
   maps feature column names to `Tensor`s containing the corresponding
   feature data, and labels is a `Tensor` containing the batch's label data.
-
+  
```

Comment:
This line has whitespace at the end of it, this one should have been trimmed.

You can configure your editor to remove trailing whitespace on save or at least to show it to you